.github/workflows/validate-new-issue-test.yml

```diff
@@ -0,0 +1,66 @@
+name: Validate new issue - test
+on:
+  workflow_dispatch:
+env:
+  GITHUB_TOKEN: ${{ secrets.PAT }}
+jobs:
+  validate-new-issue-test:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: "It skips validation for Sentry org members."
+      run: |
+        set -euo pipefail
+
+        issue_number=$(
+          basename $(
+            gh issue create --title "Greetings, program!" --body "This is an issue, clearly."
+          )
+        )
+        echo "Checking issue #${issue_number}."
+
+        function download-logs {
+          echo "Downloading and extracting the log for the validation workflow run."
+          rm -rf log.zip log
+          run_id=$(
+            gh api "repos/:owner/:repo/actions/workflows/validate-new-issue.yml/runs" \
```

Comment:
I wonder if you can get runs triggered by a specific issue, just like commits

Reply:
Did some more poking and wasn't able to find an API for that. I looked at v4 (GraphQL) as well as v3. However, I did discover a better way to pick out the run for an issue event from the list runs api: afaict the `run_number` is equal to the issue number - 1 (i.e., zero-indexed). I _think_ that should be trustworthy? 🤔

Reply:
>However, I did discover a better way to pick out the run for an issue event from the list runs api: afaict the run_number is equal to the issue number - 1 (i.e., zero-indexed). I think that should be trustworthy? 🤔

I'd rather stick to the clunky-but-documented way instead of using this unproven link.

Reply:
Moving to https://github.com/getsentry/.github/pull/24/files#r563738101 since that's the PR that introduce the `run_number` dependency.